vicodyn/src/vicodyn/proxy.cpp

```diff
@@ -22,6 +22,12 @@
 
 #include <cocaine/traits/map.hpp>
 
+namespace {
+
+static const cocaine::hpack::header_t VICODYN_ERROR_HEADER{"X-Error-Generated-By", "Vicodyn"};
```

Comment:
style: uppercase is reserved for macro. constants are declared the same way as vars

Reply:
done